validator/client/src/main/java/tech/pegasys/teku/validator/client/DefaultValidatorStatusLogger.java

```diff
@@ -170,8 +170,11 @@ public void checkValidatorStatusChanges() {
                   continue;
                 }
 
-                STATUS_LOG.validatorStatusChange(
-                    oldStatus.name(), newStatus.name(), key.toAbbreviatedString());
+                // if a validator got removed, it would have a newStatus of null
+                if (newStatus != null) {
```

Comment:
I think this is the wrong answer here.  I suspect we should iterate `newValidatorStatuses` and deal with `old` potentially being null (still report them but show them as new somehow).  That way we'll automatically stop reporting on any deleted validators but start reporting newly added ones which we're currently missing.